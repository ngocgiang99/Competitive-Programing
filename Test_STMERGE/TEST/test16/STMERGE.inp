2
1 2
889075403 756091145 
7 9
692923295 434948463 797942017 511552160 378869929 528767133 176056435 226399283 536589248 
637247221 840277492 856338363 920876281 836757791 198504377 863038975 37592743 724361214 
263867959 740194307 81150306 164216504 15346723 92704524 69022204 429096625 104366706 
553687199 142744972 737909647 460492376 942247041 137826662 946508087 94019916 268222648 
740908320 879607758 455438593 813393287 941211790 163577784 42541933 3842933 719653318 
766712218 362551493 739224902 55318812 408043099 783742846 236692479 846877989 867372830 
948640845 787354853 503489719 341078344 492189223 913650557 993890022 446825599 656539713 
