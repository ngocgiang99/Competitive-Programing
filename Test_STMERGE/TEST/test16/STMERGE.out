756091145
318015884
